tests/IntegrationTests/WebAppUiTests/TestingWebAppLocally.cs

```diff
@@ -43,7 +47,7 @@ public async Task ChallengeUser_MicrosoftIdFlow_LocalApp_ValidEmailPassword()
 
         try
         {
-            if (!UiTestHelpers.ProcessIsAlive(p)) { Assert.Fail($"Could not run web app locally."); }
+        if (!UiTestHelpers.ProcessIsAlive(p)) { Assert.Fail($"Could not run web app locally."); }
```

Comment:
nit: revert? looks like the spacing is off now

Reply:
done